crates/cheatcodes/defs/src/vm.rs

```diff
@@ -134,6 +144,49 @@ interface Vm {
         bytes stderr;
     }
 
+    /// The result of a `getStateDiff` call.
+    struct AccountAccess {
```

Comment:
Should this struct and the `StorageAccess` struct include a `forkId` and/or `chainId`, for the rationale given [here](https://github.com/foundry-rs/foundry/issues/6125#issuecomment-1799049311)? Existing recording cheats don't include this, so I'm ok not including it, but in that case we probably should create an issue track a breaking change to all these cheats that adds this (cc @Evalir)

Reply:
Just saw this comment, yes definitely agree with the inclusion of both the `forkId` and `chainId`. Even if the happy-path of using them is rarely hit, it's inexpensive to add this data to the object.